### PR TITLE
Allows catpeople to *meow

### PR DESCRIPTION
**Pull request by silicons**
https://github.com/tgstation/tgstation/issues/23489
:^)
:cl:
rscadd: Oh shit, the cats are meowing.
/:cl:

### CODE REVIEW

**Comment by Magikarpet** [comment]
make it so cat ears from the clothing machine allow you to meow

**Comment by kevinz000** [comment]
🤡

**Comment by xxalpha** [comment]
slippery slope

**Comment by jughu** [comment]
@kevinz000 only if you add them losing their voice if they spam it

**Comment by Grazyn** [comment]
As Oranges once said "This is not appropriate for the game and represent a dangerous step towards actively encouraging a known metagroup."

**Comment by Arianya** [comment]
What "known metagroup" is that?

Because I know plenty of cat-people who aren't part of some nefarious meta clique but maybe I too am part of the metagroup.

**Comment by Kromgar** [comment]
I dont want furfags

**Comment by Grazyn** [comment]
> What "known metagroup" is that?
> Because I know plenty of cat-people who aren't part of some nefarious meta clique but maybe I too am part of the metagroup.

Back at the time it was the bar-cat and the bar-assistants and their orbiters, Kevinz also mains cat as do their metabuddies (nothing bad about metabuddies anyway), not to mention that the fact itself that this emote is added for cats when lizards are still forced to custom emote their hisses means that cats are somehow privileged over elder races (flies can't buzz, plasmen can't...rattle? dunno). And if you try to balance it out you get the slippery slope which everyone hates anyway

**Comment by Arianya** [comment]
@Grazyn So you would be content if instead of an emote it was an always on speech filter similar to lizard's drawling of the letter s, which is basically a hiss anyway?

**Comment by Grazyn** [comment]
Yes I will be content if it was a speech filter like rrr, purrs/meows instead of says, or even a ~nyan added at the end of  the sentence (I can see this last one be somewhat controversial but I would be cool with it), at least it would be consistent with the other races.

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -167,3 +167,24 @@
...
+
+
+//I'm going too far.
+
+/datum/emote/sound/carbon/human/meow
+	key = "meow"
+	key_third_person = "meows"
+	message = "mewls!"
+	emote_type = EMOTE_AUDIBLE
+	sound = 'sound/effects/meow1.ogg'
```
Comment (KorPhaeron) [inline code/modules/mob/living/carbon/human/emote.dm]: no

Comment (kevinz000) [inline code/modules/mob/living/carbon/human/emote.dm]: :(

**Comment by kevinz000** [comment]
@Grazyn cats meowing is the only sound file on hand for this, get me lizard hiss fly buzz and plasma whatever and you get it. atleast, i'll make the pr for you.

**Comment by Grazyn** [comment]
On a second thought humanoids being able to spam sounds without tools doesn't seem like a great idea, but if this actually gets merged you can bet your tail I'll find those lizard sounds. I still think that giving them custom speech filter and custom say verb (meows "Hello!") would be better

**Comment by Thunder12345** [comment]
Meow :+1:

**Comment by kevinz000** [comment]
@Grazyn i'm putting a cooldown on it once i have confirmation oranges wont' come and close it immediately anyways and it's worth working on?

**Comment by Thunder12345** [comment]
Make a sperate PR putting a cooldown on all emotes, I've already had to mute borgs that spam beep and other such things

**Comment by jughu** [comment]
Put a drawback on it spamming it giving you the genetic mute disability that you lost your voice

**Comment by Arianya** [comment]
Giving speech modifications is a *little* complex for cats, because they're just defined as a human with certain (mutated) parts, rather then their own species.

You'd need to either separate them out into their own species (codewise) or introduce snowflakey checks into human species code. There's already one such snowflake check for command roles, but presumably since that's not run very often its not a key concern.

The changes needed for a catperson speech filter include a saymod (i.e. Mekhi mewls "..."), a cat tongue (every species has its own tongue for speech filters) to give them a speech filter and either splitting into own species or snowflake checks.

**Comment by KorPhaeron** [comment]
Past experience has proven that giving humans ways to spam sounds will always end in misery

**Comment by FantasticFwoosh** [comment]
Spamming meow should turn them into a cat mob after being warned to stop.

**Comment by Grazyn** [comment]
@Arianya There are already items that force a certain speech filter, just add it to the tail and ears parts

**Comment by Thunder12345** [comment]
@KorPhaeron that's what the cooldown to prevent spamming it is for

**Comment by lzimann** [comment]
stop

**Comment by Iamgoofball** [comment]
# stop

**Comment by Haevacht** [comment]
No keep going.

**Comment by optimumtact** [comment]
beat me to it kor

**Comment by Spaceinaba** [comment]
this is about as funny as a fork on a plate

**Comment by lordsaladin** [comment]
NYAAAAAAA?!?!?!?!?!!?

**Comment by Borisvanmemes** [comment]
The madman reopened it

**Pull request closed without merging**